internal/printer/printer_test.go

```diff
@@ -2815,6 +2815,15 @@ const items = ["Dog", "Cat", "Platipus"];
 				code: `${$$maybeRenderHead($$result)}<div${$$addAttribute($$renderTransition($$result, "", (slide({duration:15})), ""), "data-astro-transition-scope")}></div>`,
 			},
 		},
+		{
+			name:     "transition:animate on Component",
+			only:     true,
+			source:   `<Component class="bar" transition:animate="morph"></Component>`,
+			filename: "/projects/app/src/pages/page.astro",
+			want: want{
+				code: `${$$renderComponent($$result,'Component',Component,{"class":"bar","data-astro-transition-scope":($$renderTransition($$result, "", "morph", ""))})}`,
```

Comment:
It's tough to understand the expected result. Are the parentheses needed in `($$renderTransition($$result, "", "morph", ""))})`? 

```diff
- ($$renderTransition($$result, "", "morph", ""))
+ $$renderTransition($$result, "", "morph", "")
```

It should yield the same result

Reply:
I think our attribution expression code always wraps expressions with parens. Maybe we can change it? I don't know. I assume it's to protect against something... but what alludes me. Perhaps in case someone puts a statement inside of an expression, the parens make it into an expression? idk.